examples/ssh/iperf3.json

```diff
@@ -0,0 +1,45 @@
+{
+    "logo": "",
+    "version": "1.0.0",
+    "name": "Iperf",
+    "description": "This script is designed to retrieve network speed data using the iperf3",
+    "category": "Software Integration",
+    "requirements": {
+        "credentials": true,
+        "sandbox_version": "1.12",
+        "others": [
+            "iperf3 installed",
+            "the server iperf3 must be running"
+        ]
+    },
+    "tags": [
```

Comment:
tags are single words (mostly) and DO NOT support upper case

Please redo them
test is meaningless speedtest is ok